### DISCUSSION TITLE
Which main steps to type when launching doodba template?

### DISCUSSION THREAD

**Discussion by LudLaf** [Tecnativa/doodba#436] [Doodba - The template] [answered]
Hello there,

After typing the following in a terminal:

$ `copier copy gh:Tecnativa/doodba-copier-template docker-odoo-base-v12--10805`

and following with:

$ `cd docker-odoo-base-v12--10805`

$ `ln -s devel.yaml docker-compose.yml` # here this message shows up -> ln: failed to create symbolic link 'docker-compose.yml': File exists

$ `chown -R $USER:1000 odoo/auto`

$ `chmod -R ug+rwx odoo/auto`

$ `export UID GID="$(id -g $USER)" UMASK="$(umask)"`

$ `docker-compose -f setup-devel.yaml run --rm odoo` # and this other message here -> WARNING: Image for service odoo was built because it did not already exist. To rebuild this image you must use `docker-compose build` or `docker-compose up --build`

$ `docker-compose build`

* this line was missing from my original message:    $ `docker-compose up`

CTRL+C

$ `docker-compose down -v`

$ `docker-compose run odoo -i base --stop-after-init`

$ `docker-compose up`

In the end I manage to make it work. Then, in a browser I type: "localhost:12069"

It seems to work properly, but I wonder if there's anything missing, or probably there are some lines that shouldnt' be typed.

Any ideas to clarify and make it work better?

**Comment by pedrobaeza** [comment]
@joao-p-marques can you answer?

**Comment by joao-p-marques** [comment, marked as answer]
Hi @LudLaf 

Some of the steps you mention seem to be a part of the steps that were performed using the “old” Doodba Scaffolding. As explained [here](https://github.com/Tecnativa/doodba/discussions/67#discussioncomment-132393), that scaffolding method has been deprecated, and we are now maintaining https://github.com/Tecnativa/doodba-copier-template.

In the README in that project and in https://github.com/Tecnativa/doodba-copier-template/blob/main/docs/daily-usage.md, you should find all the information you need 😃 

As you can see from there, a lot of the commands you mention (like symlinks and chmods) are no longer needed, as they are performed as part of the `invoke` helper tasks, and in a slightly different way to avoid the errors. 

If you still find some information missing, please report back here 🙂

**Reply by LudLaf** [reply]
Hi @joao-p-marques 

Thanks for your quick response. I'll check the info, and if I get stuck again, I'll write here. 

Cheers,
